mlir/lib/Dialect/Rock/IR/TransformMapBuilder.cpp

```diff
@@ -260,7 +260,10 @@ StringRef TransformMapBuilder::endName(uint32_t dim) {
 }
 
 uint32_t TransformMapBuilder::startIndex(StringRef name) {
-  assert(startIndices.count(name) == 1 && "Key not in starting set of names");
+  if (startIndices.count(name) != 1) {
+    llvm::errs() << "Key not in starting set of names: " << name << "\n";
```

Comment:
I wanted to see *which* key wasn't in the set.

Reply:
I think we've got `report_fatal_error` down in LLVM for this sort of thing?